test/.eslintrc

```diff
@@ -0,0 +1,3 @@
+env:
```

Comment:
we can add describe as global in the main .eslintrc also ... not sure which strategy is better